plugins/vli-usbhub/fu-vli-usbhub-device.c

```diff
@@ -469,7 +471,7 @@ fu_vli_usbhub_device_erase_sector (FuVliUsbhubDevice *self, guint32 addr, GError
 	return TRUE;
 }
 
-static gboolean
+gboolean
 fu_vli_usbhub_device_erase_sectors (FuVliUsbhubDevice *self,
 				    guint32 addr,
 				    gsize sz,
```

Comment:
Should this split out to a common file?

Reply:
I don't think it can be, we need access to `self` for the things it calls.

Reply:
Ah Yeah I see.  Maybe rename the function to indicate it's operating on a common SPI in some way.

Reply:
Something like `fu_vli_usbhub_device_spi_erase`?

Reply:
ya